README.md

```diff
@@ -20,10 +20,37 @@ The site currently includes an animated landing / welcome page, functional accou
 ## Live Implemention
 To access a live prototype of the Gold Pass application please click [here](http://170.64.185.134:5000/welcome)
 
+## Getting started
+The following is a brief overview of how to navigate this code-base/how it works.
+- The three main entities are:
+    - Components
+    - Models
+    - Services 
+
+Components:
+- Components are located in /src/app/features
+- A component represents all the code (TS, HTML, SCSS) related to a given page OR part of a page
```

Comment:
Maybe reference the docs a bit, we didn't invent components or the way we used them, its just angular best practices: 
```
Components:
- [Angular Components](https://angular.io/guide/component-overview) are a convenient and modular building block for angular applications
- As per Angular best practices components are located in /src/app/feature
- each component contains the template (HTML, CSS) and functional code (TS) for its own purpose.
- Components represent either whole pages, or smaller injectable components that exist inside other pages
```